src/Models/UserModel.ts

```diff
@@ -0,0 +1,63 @@
+import { Schema, Document, model, Model } from "mongoose";
+
+export interface IUser {
+  firstname: string;
+  lastName: string;
+  fullName: string;
+  phoneNumbers?: string;
+  age?: number;
+  gender?: string;
+  email: string;
+  userDeleted?: boolean;
+  clerkId: string;
+}
+
+export interface IUserDocument extends IUser, Document {
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+const UserSchema = new Schema<IUserDocument>(
+  {
+    clerkId: {
+      type: "string",
+      required: true,
+    },
+    firstname: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    fullName: {
+      type: String,
+      required: true,
+    },
+    phoneNumbers: {
```

Comment:
I think this is a typo, but if it's not, convert the type of **"phoneNumbers"** to a string array if you are planning to store multiple phone numbers for a particular user.